development-vm/Pinfile

```diff
@@ -17,8 +17,9 @@ process :'collections-publisher' => [:'publishing-api', :'collections-publisher-
 process :'collections-publisher-worker'
 process :'contacts-admin' => [:'publishing-api', :whitehall]
 process :'content-audit-tool' => [:'publishing-api-read', :'content-audit-tool-sidekiq-google-analytics', :'content-audit-tool-sidekiq-publishing-api']
-process :'content-data-admin' =>[:'content-performance-manager', :'content-data-admin-sidekiq-default']
-process :'content-performance-manager' => [:'publishing-api', :'content-performance-manager-sidekiq-default', :'content-performance-manager-publishing-api-consumer', 'content-performance-manager-bulk-import-publishing-api-consumer', 'content-store']
+process :'content-data-admin' =>[:'content-performance-manager']
+process :'content-performance-manager' => [:'content-performance-manager-sidekiq-default']
+process :'content-performance-manager-all' => [:'publishing-api', :'content-performance-manager-sidekiq-default', :'content-performance-manager-publishing-api-consumer', 'content-performance-manager-bulk-import-publishing-api-consumer', 'content-store']
```

Comment:
I'm not quite sure what's changing here, so maybe this would be good to extract out to another commit.